mkt/ratings/views.py

```diff
@@ -100,14 +100,13 @@ def get_app(self, ident):
         except Webapp.DoesNotExist:
             raise Http404
         current_region = get_region()
-        if ((not app.is_public() or
-             not app.listed_in(region=current_region)) and
-                not check_addon_ownership(self.request, app)):
+        if (not app.is_public() and not check_addon_ownership(
```

Comment:
Now that the line is splitted after the `check_addon_ownership(` you should be able to remove the extra surrounding parenthesis